src/chrome/content/rules/University-of-California-San-Diego.xml

```diff
@@ -0,0 +1,117 @@
+		- ogs-calendar.ucsd.edu
+		- www.studentsupport.ucsd.edu
+		- vac.ucsd.edu
+
+		Peer certificate cannot be authenticated with given CA certificates:
+		- campaign.ucsd.edu
+		- giving.ucsd.edu
+		- nanoengineering.ucsd.edu
+		- pharmacy.ucsd.edu
+
+		Incomplete certificate chain error:
+		- alumni.ucsd.edu
+
+		Different content:
+		- diversity.ucsd.edu
+		- provost.ucsd.edu
+
+		Mixed content blocking (MCB) triggered:
+		- career.ucsd.edu
+		- m.ucsd.edu
+-->
+<ruleset name="University of California San Diego (partial)">
+	<target host="ucsd.edu" />
+	<target host="www.ucsd.edu" />
+
+	<target host="act.ucsd.edu" />
+	<target host="academicconnections.ucsd.edu" />
+	<target host="acms.ucsd.edu" />
+	<target host="admissions.ucsd.edu" />
+	<target host="apol-recruit.ucsd.edu" />
+	<target host="aquarium.ucsd.edu" />
+	<target host="assets.ucsd.edu" />
+
+	<target host="calendar.ucsd.edu" />
+	<target host="chemistry.ucsd.edu" />
+	<target host="cinfo.ucsd.edu" />
+	<target host="commons.ucsd.edu" />
+	<target host="courses.ucsd.edu" />
+	<target host="cse.ucsd.edu" />
+
+	<target host="disabilities.ucsd.edu" />
+	<target host="doctors.ucsd.edu" />
+
+	<target host="eaop.ucsd.edu" />
+	<target host="economics.ucsd.edu" />
+	<target host="extension.ucsd.edu" />
+
+	<target host="facilities.ucsd.edu" />
+	<target host="facultydiversity.ucsd.edu" />
+	<target host="foundation.ucsd.edu" />
+
+	<target host="git.ucsd.edu" />
+	<target host="globalhealthprogram.ucsd.edu" />
+	<target host="grad.ucsd.edu" />
+	<target host="apply.grad.ucsd.edu" />
+
+	<target host="hdh.ucsd.edu" />
+	<target host="health.ucsd.edu" />
+	<target host="healthbeat.ucsd.edu" />
+	<target host="history.ucsd.edu" />
+
+	<target host="library.ucsd.edu" />
+	<target host="libraries.ucsd.edu" />
+	<target host="libguides.ucsd.edu" />
+
+	<target host="maps.ucsd.edu" />
+	<target host="marketplace.ucsd.edu" />
+	<target host="math.ucsd.edu" />
+	<target host="mentorship.ucsd.edu" />
+	<target host="meded.ucsd.edu" />
+	<target host="mobile.ucsd.edu" />
+
+	<target host="parents.ucsd.edu" />
+	<target host="plandesignbuild.ucsd.edu" />
+	<target host="profiles.ucsd.edu" />
+	<target host="providers.ucsd.edu" />
+	<target host="psychology.ucsd.edu" />
+
+	<target host="recreation.ucsd.edu" />
+	<target host="resnet.ucsd.edu" />
+	<target host="roosevelt.ucsd.edu" />
+
+	<target host="scripps.ucsd.edu" />
+	<target host="software.ucsd.edu" />
+	<target host="structures.ucsd.edu" />
+	<target host="www.structures.ucsd.edu" />
+	<target host="students.ucsd.edu" />
+	<target host="studentsupport.ucsd.edu" />
+	<target host="sustainability.ucsd.edu" />
+
+	<target host="ted.ucsd.edu" />
+	<target host="transportation.ucsd.edu" />
+	<target host="tritoned.ucsd.edu" />
+
+	<target host="ucsdnews.ucsd.edu" />
+	<target host="ugcbo.ucsd.edu" />
+
+	<target host="wellness.ucsd.edu" />
+	<target host="women.ucsd.edu" />
+
+	<rule from="^http:" to="https:" />
+</ruleset>
```

Comment:
The new ruleset has a significant worse coverage than the old one.
I dodn't check the nonfunctional subdomains, but your list is also significant shorter.

Here are some targets that are missing in the new ruleset:
```
a.ucsd.edu
a4.ucsd.edu (`http://a4.ucsd.edu/tritON/`)
academicconnections.ucsd.edu
acms.ucsd.edu
acs-webmail.ucsd.edu
act.ucsd.edu
admissions.ucsd.edu
apol-recruit.ucsd.edu
apply.grad.ucsd.edu
aquarium.ucsd.edu
artsandhumanities.ucsd.edu
as.ucsd.edu (cert-chain)
assets.ucsd.edu
biology.ucsd.edu
blink.ucsd.edu
caesar.ucsd.edu
calendar.ucsd.edu
career.ucsd.edu
cfr.ucsd.edu
cgs.ucsd.edu
chd.ucsd.edu
chemistry.ucsd.edu
cilas.ucsd.edu
cinfo.ucsd.edu
commons.ucsd.edu
courses.ucsd.edu
create.ucsd.edu
crlpsandiego.ucsd.edu
cse.ucsd.edu
cseweb.ucsd.edu
csp.ucsd.edu
cwo.ucsd.edu
dah.ucsd.edu
dbsportal.ucsd.edu
desktop.ucsd.edu
developer.ucsd.edu
disabilities.ucsd.edu
divelog.ucsd.edu
doctors.ucsd.edu
eaop.ucsd.edu
economics.ucsd.edu
ethnicstudies.ucsd.edu
extension.ucsd.edu
facilities.ucsd.edu
facultydiversity.ucsd.edu
foundation.ucsd.edu
git.ucsd.edu
globalhealthprogram.ucsd.edu
grad.ucsd.edu
hdh.ucsd.edu
hds.ucsd.edu
health.ucsd.edu
healthbeat.ucsd.edu
healthsciences.ucsd.edu
history.ucsd.edu
hsreservations.ucsd.edu
igpp.ucsd.edu
igppticket.ucsd.edu
igppwiki.ucsd.edu
isp.ucsd.edu
iwdc.ucsd.edu
jobs.ucsd.edu
libguides.ucsd.edu
libraries.ucsd.edu
library.ucsd.edu
lists.ucsd.edu
m.ucsd.edu
mailman.ucsd.edu
maps.ucsd.edu
marketplace.ucsd.edu
math.ucsd.edu
mathlink.ucsd.edu
mathlink2.ucsd.edu
meded.ucsd.edu
mentorship.ucsd.edu
mobile.ucsd.edu
myextension.ucsd.edu
myosd.ucsd.edu
mytritonlink.ucsd.edu
myucsdchart.ucsd.edu
ocfr.ucsd.edu
ocga.ucsd.edu
onthego.ucsd.edu
pao.ucsd.edu
parents.ucsd.edu
physics.ucsd.edu
plandesignbuild.ucsd.edu
podcast.ucsd.edu
polisci.ucsd.edu
preuss.ucsd.edu
profiles.ucsd.edu
providers.ucsd.edu
psychology.ucsd.edu
recreation.ucsd.edu
resnet.ucsd.edu
rmp.ucsd.edu
roger.ucsd.edu
roosevelt.ucsd.edu
saber.ucsd.edu
sarc.ucsd.edu
sciencestudies.ucsd.edu
scripps.ucsd.edu
sdacs.ucsd.edu
sdawp.ucsd.edu
se.ucsd.edu
ships.ucsd.edu
shs.ucsd.edu
socialsciences.ucsd.edu
software.ucsd.edu
structures.ucsd.edu
student.ucsd.edu
studentparents.ucsd.edu
students.ucsd.edu
studentsupport.ucsd.edu
sustainability.ucsd.edu
sysstaff.ucsd.edu
ted.ucsd.edu
transportation.ucsd.edu
trio.ucsd.edu
tritoned.ucsd.edu
tritonlink.ucsd.edu
uclearning.ucsd.edu
ucsdbkst.ucsd.edu
ucsdfoundation.ucsd.edu
ucsdnews.ucsd.edu
ugcbo.ucsd.edu
ui.ucsd.edu
unex-shibboleth.ucsd.edu
usmex.ucsd.edu
usp.ucsd.edu
uxe.ucsd.edu
wellness.ucsd.edu
women.ucsd.edu
wts.ucsd.edu
www-acs.ucsd.edu
www-act.ucsd.edu
www-er.ucsd.edu
www-structures.ucsd.edu
www.structures.ucsd.edu
www.ucsd.edu
yankelovichcenter.ucsd.edu
```